dotcom-rendering/docs/contributing/how-to.md

```diff
@@ -25,3 +37,41 @@ If you need JavaScript on the client, there are two main options:
 
 Both of these approaches should be used with caution, though. Feel free to reach
 out to the team if you're working on a feature in DCR which needs client-side scripts!
+
+## How can I create an 'island' in DCR?
+
+The 'islands' pattern is DCR's way of adding client-side React code to the site.
+You can read more about it in our
+[Improved Partial Hydration](architecture/027-better-partial-hydration.md)
+decision document.
+
+To add an island:
+
+1. Wrap your component on the server with an `<Island>` component.
+2. Add `.importable` to the component filename. Eg: `[MyThing].importable.tsx`
+3. Specify what should trigger hydration (e.g. waiting until the component
+  scrolls into view). See `Island.tsx` props for options.
+
+## How can I make an AJAX request in DCR?
```

Comment:
Not really sure I would call it AJAX as we don't use that underlying tech anymore.

```suggestion
## How do fetch external data on the client?
```

Or something?

Reply:
TIL!

I thought that AJAX was basically interchangeable with 'client-side HTTP requests launched via javascript'; didn't realise it meant something more specific. Happy to update as suggested!